curtisallen/turningpoint-cms
# Simplify landingpage

#### What's this PR do?

Simplifies the landing page, removes some of the default keystone stuff. 
#### Where should the reviewer start?

`templates/views/index.jade`

#### How should this be manually tested?
Start the app 

```console 
npm install
gulp startMongo
gulp
```

Then navigate to https://localhost:3000

You should see something like
<img width="1230" alt="screen shot 2017-03-07 at 9 41 41 pm" src="https://cloud.githubusercontent.com/assets/270735/23690443/ead70546-037e-11e7-977f-900831afdd81.png">
